commands/bot.go

```diff
@@ -216,22 +217,27 @@ func botListCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 func botEnableCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	users := getUsersFromUserArgs(c, args)
+
+	var result *multierror.Error
+
 	for i, user := range users {
 		if user == nil {
 			printer.PrintError(fmt.Sprintf("can't find user '%v'", args[i]))
+			result = multierror.Append(result, fmt.Errorf("can't find user '%v'", args[i]))
 			continue
 		}
 
 		bot, _, err := c.EnableBot(user.Id)
 		if err != nil {
 			printer.PrintError(fmt.Sprintf("could not enable bot '%v'", args[i]))
+			result = multierror.Append(result, fmt.Errorf("could not enable bot '%v'", args[i]))
```

Comment:
Same here. Please also include the error with `%w`.
```suggestion
			result = multierror.Append(result, fmt.Errorf("could not enable bot %q: %w", args[i], err))
```